code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -65,6 +65,9 @@
 	projectile_type = "/obj/item/projectile/bullet/midbullet"
 	w_type = RECYK_METAL
 
+/obj/item/ammo_casing/a12mm/assault
+	projectile_type = "/obj/item/projectile/bullet/assault"
```

Comment:
This path is incorrect. It should be `/obj/item/projectile/bullet/midbullet/assault`.